pkg/manager/manager.go

```diff
@@ -478,3 +495,21 @@ func (m *manager) getUnusedVlanList(brif []netlink.Link, vlans []int) (unusedVla
 
 	return unusedVlans
 }
+
```

Comment:
nit: could you move these after L37 so they are placed in the same vicinity as the struct definition ?
just so its easier to find related vlanLock methods